220080/문제3.py

```diff
@@ -0,0 +1,15 @@
+# 소득 불균형
+
+import sys
+sys.stdin = open('input.txt','r')
+
+T = int(input())
+for t in range(1,T+1):
+    cnt =0
+    num = int(input())
+    num1 = list(map(int,input().split()))
+    b =sum(num1)//num
```

Comment:
깔끔하네요! 😄 
```suggestion
    b =sum(num1)/num
```
추가로, 정확한 평균값은 `//`가 아닌 `/`로 구하는 게 맞습니다!